packages/lit-dev-content/site/blog/2022-10-13-lit-labs-event.md

```diff
@@ -0,0 +1,38 @@
+---
+tags: blogPosts
+layout: blog-post.html
+title: "Get the low-down on Lit Labs"
+summary: "We are having a virtual event on October 24th 2022 at 10 AM PST to talk about the status of Lit Labs!"
+date: 2022-10-13
+author:
+  - elliott-marquez
+eventDate: "Monday October 24th 2022 at 10AM PDT"
+ytLink: https://www.youtube.com/
+---
+
+The Lit team is hosting a virtual event on **{{eventDate}}** to talk about all of the exciting work going on in Lit Labs. Join us for detailed presentations on the current and future offerings of our Labs packages, their statuses, and how you can help get them across the line and shape the future of Lit!
+
+The event will be hosted [**at this link**]({{ytLink}}) on YouTube. Make sure to sign up for reminders and notifications so you don't miss it!
+
+<img src="/images/blog/labs-event/lit-labs.svg" width="720" class="block centered">
+
+## What is Lit Labs?
+
+[Lit Labs]({{site.baseurl}}/docs/libraries/labs/) is an umbrella for projects related to Lit that aren't quite ready for production. These projects may be experimental or incomplete, or simply looking for real-world feedback. Lit Labs projects are published under the [`@lit-labs` npm scope](https://www.npmjs.com/search?q=%40lit-labs).
+
+## What will be covered at the Lit Labs event?
+
+The Lit team will be covering information on how individual Labs projects can help you in your Web Component projects as well as their current status for Labs graduation into stable. Here is a list of the projects that will be covered:
+
+- [Task](https://github.com/lit/lit/tree/main/packages/labs/task)
+- [Virtualizer](https://github.com/lit/lit/tree/main/packages/labs/virtualizer)
+- [Motion](https://github.com/lit/lit/tree/main/packages/labs/motion)
+- [Context](https://github.com/lit/lit/tree/main/packages/labs/context)
+- [React](https://github.com/lit/lit/tree/main/packages/labs/react)
+- [CLI](https://github.com/lit/lit/tree/main/packages/labs/cli)-generated Framework Wrappers
+- [SSR](https://github.com/lit/lit/tree/main/packages/labs/ssr)
+- [Observers](https://github.com/lit/lit/tree/main/packages/labs/observers)
+
+## When is it?
+
+The event will be on **{{eventDate}}**. Make sure to sign up for YouTube reminders and notifications [**at this link**]({{ytLink}}) so you don't miss it!
```

Comment:
As above ...  "at this link" ==> "on the event page", "for the live stream event," or something like that.